src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,685 @@
+          int7: {^RandomInt: {min: -100, max: 100}}
+          int8: {^RandomInt: {min: -100, max: 100}}
+          int9: {^RandomInt: {min: -100, max: 100}}
+          int10: {^RandomInt: {min: -100, max: 100}}
+          str1: {^RandomString: {length: 16}}
+          str2: {^RandomString: {length: 16}}
+          str3: {^RandomString: {length: 16}}
+          str4: {^RandomString: {length: 16}}
+          str5: {^RandomString: {length: 16}}
+          str6: {^RandomString: {length: 16}}
+          str7: {^RandomString: {length: 16}}
+          str8: {^RandomString: {length: 16}}
+          str9: {^RandomString: {length: 16}}
+          str10: {^RandomString: {length: 16}}
+          arr1: &arr1 [{^Repeat: {count: 10, fromGenerator: {b: 1, c: 1}}}]
+          arr1Dup1: *arr1
+          arr1Dup2: *arr1
+          arr1Dup3: *arr1
+          arr1Dup4: *arr1
+          arr1Dup5: *arr1
+          arr1Dup6: *arr1
+          arr1Dup7: *arr1
+          arr1Dup8: *arr1
+          arr1Dup9: *arr1
+          arr2: [{^Repeat: {count: 10, fromGenerator: {b: 2, c: 2}}}]
+          arr3: [{^Repeat: {count: 10, fromGenerator: {b: 3, c: 3}}}]
+          arr4: [{^Repeat: {count: 10, fromGenerator: {b: 4, c: 4}}}]
+          arr5: [{^Repeat: {count: 10, fromGenerator: {b: 5, c: 5}}}]
+          arr6: [{^Repeat: {count: 10, fromGenerator: {b: 6, c: 6}}}]
+          arr7: [{^Repeat: {count: 10, fromGenerator: {b: 7, c: 7}}}]
+          arr8: [{^Repeat: {count: 10, fromGenerator: {b: 8, c: 8}}}]
+          arr9: [{^Repeat: {count: 10, fromGenerator: {b: 9, c: 9}}}]
+          arr10: [{^Repeat: {count: 10, fromGenerator: {b: 10, c: 10}}}]
+
+# Phase 2: Ensure all data is synced to disk.
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+# Phase 3-N: Run variadic aggregate expression benchmarks
+- ExpressionAddFive:
+  LoadConfig: &loadConfig
+    Path: "../../phases/execution/AggregateExpressions.yml"
+    Key: ExpressionTemplate
+    Parameters:
+      name: VariadicExpressionAddFive
+      nopInPhasesUpTo: *MaxPhases
+      active: [3]
+      operationMetricsName: VariadicAggExpressionAddFive
+      Pipeline: [{$project: {add: {$add: *FiveIntFields}}}]
+
+- ExpressionAddTen:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionAddTen
+      nopInPhasesUpTo: *MaxPhases
+      active: [4]
+      operationMetricsName: VariadicAggExpressionAddTen
+      Pipeline: [{$project: {add: {$add: *TenIntFields}}}]
+
+- ExpressionAddFifty:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionAddFifty
+      nopInPhasesUpTo: *MaxPhases
+      active: [5]
+      operationMetricsName: VariadicAggExpressionAddFifty
+      Pipeline: [{$project:
+                    {add:
+                       {$add:
```

Comment:
discussing offline how we might simplify this further

Reply:
As per offline discussion, I made changes to the generator by adding a deterministic mode to Choose, and now we just control the parameter "number" when constructing an array.

Reply:
Nice work on this; resolving!